Add windows documentation (#590)
codemagic-ci-cd/codemagic-docs: content/yaml-publishing/distribution.md

Post-processing does not consume any build minutes.

### Microsoft Partner Center

{{<notebox>}}
DISCLAIMER: Building Windows applications with Codemagic is in beta stages and will be free to all users with billing enabled until the end of 2021. Any feedback is well received in our [Slack channel](https://slack.codemagic.io).
{{</notebox>}}

> Codemagic enables you to automatically publish your desktop app to the Microsoft Store. Note that a prior successful submission has to have been made through the Partner Center in order for Codemagic to publish your application.

Reviewer: ```suggestion
Codemagic enables you to automatically publish your Windows desktop app to the Microsoft Store. Note that a prior successful submission through the Partner Center is required for Codemagic to be able to publish your application.
```

Reviewer: Add the same note here too that the msix package can be downloaded from build artifacts and uploaded manually

> For publishing, Codemagic makes use of the [Microsoft Store submission API](https://docs.microsoft.com/en-us/windows/uwp/monetize/create-and-manage-submissions-using-windows-store-services). Thus it is also necessary to link your Partner Center account to an Azure AD application and providing Codemagic with information that can be used to generate temporary Azure AD access tokens for managing submissions.

Reviewer: ```suggestion
For publishing, Codemagic makes use of the [Microsoft Store submission API](https://docs.microsoft.com/en-us/windows/uwp/monetize/create-and-manage-submissions-using-windows-store-services). This requires linking your Partner Center account to the Azure AD application and providing Codemagic with information that can be used to generate temporary Azure AD access tokens for managing submissions.
```

> To get the necessary details (`tenant_id`, `client_id`, `client_secret`), follow the instructions [here](../knowledge-base/partner-center-authentication).

Reviewer: ```suggestion
To link your Microsoft Partner Center account with the Azure AD application and get the necessary details (`tenant_id`, `client_id`, `client_secret`), follow the instructions [here](../knowledge-base/partner-center-authentication).
```

> It is also necessary for you to provide your `store_id`, which can be found in the [Partner Center apps dashboard URL](https://partner.microsoft.com/en-us/dashboard/apps), when looking at the details for your applications.

Reviewer: ```suggestion
It is also necessary for you to provide your `store_id`, which can be found when you open the application in the [Partner Center apps dashboard](https://partner.microsoft.com/en-us/dashboard/apps).

> To safely store and use the `client_secret`, configure it as an [environment variables](/variables/environment-variable-groups/#storing-sensitive-valuesfiles) in Codemagic. Click **Secure** to encrypt the value.

Reviewer: ```suggestion
To safely store and use the `client_secret`, save it as an [environment variable](/variables/environment-variable-groups/#storing-sensitive-valuesfiles) in the Codemagic UI. Click **Secure** to encrypt the value.
```
